arbnode/espresso_batch_poster_helpers.go

```diff
@@ -0,0 +1,95 @@
+package arbnode
```

Comment:
move this file to this folder that Theo has created - https://github.com/EspressoSystems/nitro-espresso-integration/tree/integration/espresso

Reply:
This is a file containing our methods in the batch poster. It is not a standalone package.

I was trying to follow this file: https://github.com/EspressoSystems/nitro-espresso-integration/blob/8877aab451f510bf756267c7dc8e6714072ecf6a/arbnode/espresso_transaction_streamer_helpers.go#L1